doc/nrf/ug_matter_creating_accessory.rst

```diff
@@ -250,76 +255,72 @@ To add a new event handler, complete the following steps:
               }
       }
 
-Include headers for clusters' callbacks
-=======================================
+Include header for cluster attribute helpers
+============================================
 
-To have the required callback declaration for the Temperature Measurement cluster, make sure to include the following file in the :file:`src/app_task.cpp` file:
+To import helper functions for accessing cluster attributes, make sure to include the following file in the :file:`src/app_task.cpp` file:
 
 .. code-block:: C++
 
-   #include <app/clusters/temperature-measurement-server/temperature-measurement-server.h>
+   #include <app-common/zap-generated/attributes/Accessors.h>
 
 .. rst-class:: numbered-step
 
 Create a callback for sensor activation and deactivation
 ********************************************************
 
-Up until now the events :c:struct:`SensorActivate` and :c:struct:`SensorDeactivate` are not present in the event queue.
-This is because the attribute of the On/off cluster is not changed from within the ``AppTask``.
-It can only be changed externally, for example using the Matter controller.
+Handlers for the :c:struct:`SensorActivate` and :c:struct:`SensorDeactivate` events are now ready, but the events are not posted to the event queue.
+The sensor is supposed to be turned on and off remotely by changing the ``OnOff`` attribute of the On/off cluster, for example using the Matter controller.
+This means that we need to implement a callback function to post one of these events every time the ``OnOff`` attribute changes.
 
-However, the sensor application needs to know how this attribute is changing to know when the sensor is turned on and off.
-This means that there is a need to implement a callback function to post one of these events every time the On/off attribute changes.
+To implement the callback function, complete the following steps:
+1. Create a new file, for example :file:`src/zcl_callbacks.cpp`.
```

Comment:
We need a blank line before this line. Otherwise, the list will be a continuation of the previous sentence.

Reply:
I need to find some tool to show somewhat real-time preview of these doc changes as I can't get used to this syntax :D